administrator/language/de-DE/joomla.ini

```diff
@@ -447,11 +472,12 @@ JGLOBAL_INHERIT="Vererbt"
 JGLOBAL_INTEGRATION_LABEL="Integration"
 JGLOBAL_INTRO_TEXT="Einleitungstext"
 JGLOBAL_ISFREESOFTWARE="%s ist freie, unter der <a href=\"https://www.gnu.org/licenses/gpl-2.0.html\" target=\"_blank\" rel=\"noopener noreferrer\">GNU/GPL-Lizenz</a>, veröffentlichte Software."
+JGLOBAL_ITEM_FEATURE="Element darstellen"
+JGLOBAL_ITEM_UNFEATURE="Element ausblenden"
 JGLOBAL_KEEP_TYPING="Weiter tippen ..."
 JGLOBAL_LANGUAGE_VERSION_NOT_PLATFORM="Das Sprachpaket passt nicht zur installierten Joomla!-Version. Einige Übersetzungen könnten daher fehlen und werden in Englisch angezeigt."
```

Comment:
@max123kl bitte mal Prüfen ob dies im Core noch mit ! geschrieben wird aktuell.

Reply:
Im letzten Commit habe ich die Reihenfolge der Übersetzungs-Strings exakt auf die Core-Version abgestimmt.
Die Source-Version ist in der Commit-Nachricht vermerkt.

Wenn keine anderen Gründe dagegen sprechen, kann man deshalb ältere Issues/PRs, soweit sie diese Datei betreffen, vernachlässigen. Derzeit offene Issues/PRs habe daher ich nicht berücksichtigt.  

Die Schreibweise in der Datei joomla.ini ist uneinheitlich in Zeile 258:  
`JFIELD_NAME_DESC="The name will be used to identify the field. Leave this blank and Joomla will fill in a default value from the title."`

in Zeile 474 dagegen:
`JGLOBAL_LANGUAGE_VERSION_NOT_PLATFORM="Language pack does not match this Joomla! version. Some strings may be missing and will be displayed in English."`

In den Richtlinien für [JDOC](https://docs.joomla.org/JDOC:Words_to_watch) steht:
`Joomla! and Joomla  —  Always capitalise and include the exclamation mark on the first usage on a page (usually a title). Otherwise, like in code, capitalise and don't include the exclamation mark.`

Allerdings ist die Datei vor mehr als 4 Jahren zuletzt aktualisiert worden. Momentan habe ich keine andere Information.

Reply:
https://docs.joomla.org/Joomla:Proper_use_of_the_Joomla!_name#
> Whenever the Joomla! name is part of or nested inside another multi-word Joomla-related trademark or service mark then the exclamation point should be used. If the use is a proper name it should have capitalised initials, Joomla! should include the exclamation point and it should be followed by a TM indicia. Examples: References to the Joomla! Extensions Directory™, the Joomla! Framework™ or Joomla! 3.4 as a product.

Es bleibt hier bei: Joomla!-Version